### PR TITLE
add migration for removing AcceptedTermsOfServiceId column from Users

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Cherry-pick  #19263 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by streamer45 — APPROVED** [review]
:100:

File: store/sqlstore/upgrade.go [hunk 1]
```diff
@@ -1827,3 +1829,13 @@ func upgradeDatabaseToVersion630(sqlStore *SqlStore) {
...
 	}
 }
+
+func upgradeDatabaseToVersion631(sqlStore *SqlStore) {
+	// if shouldPerformUpgrade(sqlStore, Version630, Version631) {
+	if sqlStore.DoesColumnExist("Users", "AcceptedTermsOfServiceId") {
+		sqlStore.GetMaster().ExecNoTimeout("ALTER TABLE Users DROP COLUMN AcceptedTermsOfServiceId")
+	}
+
+	// saveSchemaVersion(sqlStore, Version631)
```
Comment (streamer45) [inline store/sqlstore/upgrade.go:1839]: We may want to uncomment this.

**Comment by isacikgoz** [comment]
@amyblais We discussed this offline and since we haven't shipped the 6.3, we thought that it would be safe to include this in 6.3 migrations. WDYT?

**Comment by amyblais** [comment]
> @amyblais We discussed this offline and since we haven't shipped the 6.3, we thought that it would be safe to include this in 6.3 migrations. WDYT?

Sounds good.

**Pull request merged**